fact_extractor/install/common.py

```diff
@@ -4,36 +4,18 @@
 from pathlib import Path
 
 from helperFunctions.config import load_config
-from helperFunctions.install import (
-    apt_install_packages, apt_update_sources, pip3_install_packages
-)
+from helperFunctions.install import apt_install_packages, apt_update_sources, pip_install_packages
 
 
 DEPENDENCIES = {
     # Ubuntu
-    'xenial': {},
-    'bionic': {
-        'pip3': [
-            'testresources'
-        ]
-    },
-    'focal': {
-        'pip3': [
-            'testresources'
-        ]
-    },
+    'bionic': {},
+    'focal': {},
+    'jammy': {},
     # Debian
-    'buster': {
-        'pip3': [
-            'testresources'
-        ]
-    },
-    'bullseye': {
-        'pip3': [
-            'testresources'
-        ]
-    },
-    # Packages common to all plateforms
+    'buster': {},
+    'bullseye': {},
```

Comment:
We will currently never get to bullseye, as it is not in the supported list in `fact_extractor/install.py`, correct? If so, we can remove all bullseye specific assignments here as well.

Reply:
actually it is included in `fact_extractor/install.py`. I also added it in the exit message. Also why remove bullseye and keep buster when bullseye is newer?

Reply:
There was also some rudimentary support for Kali but it was included in the buster codenames. I changed it to bullseye (which makes a bit more sense IMHO as it's base is probably newer than buster)